jetty-util/src/main/java/org/eclipse/jetty/util/ssl/SslContextFactory.java

```diff
@@ -1761,10 +1791,14 @@ public SSLParameters customize(SSLParameters sslParams)
             sslParams.setCipherSuites(_selectedCipherSuites);
         if (_selectedProtocols != null)
             sslParams.setProtocols(_selectedProtocols);
-        if (getWantClientAuth())
-            sslParams.setWantClientAuth(true);
-        if (getNeedClientAuth())
-            sslParams.setNeedClientAuth(true);
+        if (this instanceof Server)
```

Comment:
This unfortunately breaks compatibility with code originally written against earlier Jetty versions:
Such code instantiates an instance of `SslContextFactory` (and not a subclass thereof) which doesn't satisfies this type check and as a result no longer has the two client auth options set for its server.

Reply:
Doh, good catch! Thanks!